how-to/maintain/identify-abnormal-queries/identify-expensive-queries.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: 定位消耗系统资源多的查询
 category: how-to
+aliases: ['/docs-cn/how-to/maintain/identify-abnormal-queries/identify-aborted-queries/']
```

Comment:
expensive ?

Reply:
https://github.com/pingcap/docs/pull/1804 The file name in this PR is `identify-aborted-queries`, but i t should be `identify-expensive-queries`.